packages/components/components/elvis-popover/src/react/useReturnFocus.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+
+export const useReturnFocus = (isOpen: boolean) => {
+  /* Saving the original focused element before the popover is opened, and then returning focus to that
+  element when the popover is closed. */
+  useEffect(() => {
+    if (!isOpen) {
+      return;
+    }
+    const originalFocusedElement = document.activeElement as HTMLElement | null;
+    return () => {
+      originalFocusedElement?.focus();
+    };
```

Comment:
Maybe this part could be baked into the `useFocusTrap`? It makes sense that it also cleans up after itself.

Reply:
Ooh I like that idea yeah!